packages/react/src/components/UIShell/UIShell.HeaderBase.stories.js

```diff
@@ -437,7 +443,9 @@ export const HeaderWSideNav = () => (
             aria-label="Side navigation"
             expanded={isSideNavExpanded}
             onSideNavBlur={onClickSideNavExpand}
-            href="#main-content">
+            href="#main-content"
+            inert={isSideNavExpanded ? false : -1} // With the `inert` prop you can disable the navigation from all children elements
+          >
```

Comment:
If `isSideNavExpanded` is already given to SideNav via `expanded`, would it make more sense for `inert` to be conditionally applied _within_ SideNav? Otherwise we're leaving this up to the consumer to properly implement, which is unlikely

Reply:
Defenitily! Idk why I didn't that before 🤔